src/views/Stake/Stake.jsx

```diff
@@ -423,31 +426,32 @@ function Stake() {
                     <TableRow>
                       <TableCell>
                         <Box className="ohm-pairs">
-                          <BondLogo bond={ohm_frax}></BondLogo>
-                          <Typography>OHM-FRAX</Typography>
+                          <BondLogo bond={{ bondIconSvg: OhmLusdImg, isLP: true }}></BondLogo>
```

Comment:
Can you just pass in the bond object?

Reply:
We don't have an Ohm-Lusd bond yet. When we do I'll pass the bond object.